api/src/organizational-entities/application/certification-center/certification-center.admin.route.js

```diff
@@ -121,6 +121,34 @@ const register = async function (server) {
         tags: ['api', 'admin', 'certification-center'],
       },
     },
+    {
+      method: 'POST',
+      path: '/api/admin/certification-centers/{id}/archive',
```

Comment:
Nommer explicitement les paramètres de route pour du code plus clair et faciliter la factorisation de preHandler : 
```suggestion
      path: '/api/admin/certification-centers/{certificationCenterId}/archive',
```